DependencyInjection/MassiveSearchExtension.php

```diff
@@ -36,7 +36,19 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadLocalization($config, $loader, $container);
         $this->loadSearch($config, $loader, $container);
-        $this->loadMetadata($config, $loader, $container);
+        $this->loadMetadata($loader, $container);
+        $this->loadPersistence($config['persistence'], $loader);
+    }
+
+    private function loadPersistence($config, Loader\XmlFileLoader $loader)
+    {
+        foreach ($config as $persistenceName => $config) {
+            if (false === $config['enabled']) {
+                return;
```

Comment:
Shouldn't this be continue? Otherwise it will stop for the first configuration which is not enabled.